### ISSUE TITLE
Avoid hard-coding dom0 in `qubes-video-companion` script

### ISSUE THREAD

**Issue by DemiMarie** [elliotkillick/qubes-video-companion#15]
On my system, webcams are attached to `sys-usb`.  One alternative is to specify the target in qrexec policy instead.  I will make a PR.

**Comment by elliotkillick** [comment]
Yeah, I wasn't quite sure how `qrexec-client-vm` would work for specifying multiple possible target qubes as it only allowed for specifying one. I figured if I just specify `dom0` then because I've installed the qrexec policies (in the RPM package) in dom0 that should work for `sys-usb` (there is a policy added to allow `sys-usb`) too.

Is this assumption incorrect or is this alternative you're talking about just a better way of doing this?

**Comment by pietrushnic** [comment]
I have the same problem using v1.0.0-1 deb
